Shared/SubProcessHost.cs

```diff
@@ -20,7 +20,8 @@ namespace OpenTap
     class SubProcessHost
     {
         public bool ForwardLogs { get; set; } 
-        public string LogHeader { get; set; } = "Subprocess";
+        public string LogHeader { get; set; } = "";
+        public string[] MutedSources { get; set; } = Array.Empty<string>();
```

Comment:
maybe this should be a `HashSet<string>`

Reply:
Hmm. I agree.